### PR TITLE
Request attestation endpoint implemented

### PR DESCRIPTION
**Pull request by SIKHAR405**
- implemented request attestation endpoint under local operations for iin agent
- implemented some minor bug fixes in localserver.ts and server.ts
- added test for 2 iin agents in workflow and implemented some minor bug fixes in the test for 2 iin agents
- minor updates to the docker-testnet environment files
- added dns config file involving 2 iin agents
- minor updates to the documentation for installing the Go code generator plugins for protobuf-compiler

### CODE REVIEW

File: samples/fabric/fabric-cli/src/commands/configure/network.ts [hunk 1]
```diff
@@ -54,6 +59,10 @@ const command: GluegunCommand = {
       logger.level = 'debug'
       logger.debug('Debugging is enabled')
     }
+    let members = [global.__DEFAULT_MSPID__]
+    if (options["num-orgs"] === 2){
+      members = ["Org1MSP","Org2MSP"]
```
Comment (VRamakrishna) [inline samples/fabric/fabric-cli/src/commands/configure/network.ts:64]: Same comment as above.

File: samples/fabric/fabric-cli/src/commands/configure/all.ts [hunk 2]
```diff
@@ -58,6 +64,10 @@ const command: GluegunCommand = {
       logger.level = 'debug'
       logger.debug('Debugging is enabled')
     }
+    let members = [global.__DEFAULT_MSPID__]
+    if (options["num-orgs"] === 2){
+      members = ["Org1MSP","Org2MSP"]
```
[the remark this answers is not in the document]
Comment (SIKHAR405) [reply samples/fabric/fabric-cli/src/commands/configure/all.ts:69]: Addressed this.

File: samples/fabric/fabric-cli/src/commands/configure/network.ts (hunk 1 above)
Comment (SIKHAR405) [reply samples/fabric/fabric-cli/src/commands/configure/network.ts:64]: Addressed this.

**Comment by SIKHAR405** [comment]
Created new PR #331 with same changes to resolve the DCO issue. Closing the current PR.

**Pull request closed without merging**